packages/calcite-components/src/components/combobox-item/combobox-item.scss

```diff
@@ -64,9 +61,9 @@ ul:focus {
   ease-in-out;
   @include word-break();
   justify-content: space-around;
-  gap: var(--calcite-combobox-item-spacing-unit-l);
+  gap: var(--calcite-combobox-item-spacing-indent);
   padding-block: var(--calcite-combobox-item-spacing-unit-s);
-  padding-inline: var(--calcite-combobox-item-spacing-unit-l);
+  padding-inline-end: var(--calcite-combobox-item-spacing-indent);
   padding-inline-start: var(--calcite-combobox-item-indent-value);
```

Comment:
nitpick: difference between `indent` and `indent-value`? Maybe these could be renamed as well.